haystack/urls.py

```diff
@@ -1,16 +1,12 @@
 # encoding: utf-8
 
-from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
This doesn't seem like a necessary change. I'd like to leave that in so we have a consistent __future__ policy across every file.